benchopt/plotting/generate_html.py

```diff
@@ -230,24 +230,27 @@ def shape_solvers_for_html(df, objective_column):
 
         color, marker = get_solver_style(solver)
         # to preserve support of previous benchopt version
-        # where 'stopping_strategy' wasn't saved in solver meta
+        # where 'sampling_strategy' wasn't saved in solver meta
         try:
-            stopping_strategy = df_filtered['stopping_strategy'].unique()
+            sampling_strategy = df_filtered['sampling_strategy'].unique()
         except KeyError:
-            stopping_strategy = ["Time"]
-
-        if len(stopping_strategy) != 1:
-            found_stopping_strategies = ', '.join(
-                f"`{item}`" for item in stopping_strategy
+            try:
+                sampling_strategy = df_filtered['stopping_strategy'].unique()
```

Comment:
@tomMoral should we preserve backwards compatibility with all parquet/csv? we didn't when we changed from `stop_strategy` to `stopping_strategy`

Reply:
Yes we should but it will be bothersome...